meson.build

```diff
@@ -32,6 +32,19 @@ if cc.get_id() == 'clang' or cc.get_id() == 'gcc'
   endif
 endif
 
+# Use git short revision as build identifier.
+git = find_program('git', required: false)
+if git.found()
+  r = run_command(git, 'rev-parse', '--short', 'HEAD')
+  if r.returncode() == 0
+    short_rev = r.stdout().strip()
+    message('found git short revision ' + short_rev)
+    add_project_arguments('-DBUILD_IDENTIFIER="g' + short_rev + '"', language : 'cpp')
+  endif
+else
+  warning('Please install git to add short revision to build identifier in version string.')
```

Comment:
What about making `short_rev` "unknown" in this case?
That being said, I found (the hard way) that without git meson fails when trying to get a subproject, so I have already hardcoded git use in <https://github.com/LeelaChessZero/lc0/blob/master/meson.build#L69>.

Reply:
Done.